playbooks/azure/openshift-cluster/group_vars/all/yum_repos.yml

```diff
@@ -33,16 +33,16 @@ azure_node_repos:
     sslclientkey: /var/lib/yum/client-key.pem
     enabled: yes
 
-  - name: rhel-7-server-ansible-2.4-rpms
-    baseurl: https://mirror.openshift.com/enterprise/rhel/rhel-7-server-ansible-2.4-rpms/
+  - name: rhel-7-server-ansible-2.6-rpms
+    baseurl: https://mirror.openshift.com/enterprise/rhel/rhel-7-server-ansible-2.6-rpms/
     gpgkey: file:///etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release
     sslclientcert: /var/lib/yum/client-cert.pem
     sslclientkey: /var/lib/yum/client-key.pem
     enabled: yes
 
-  # TODO: Replace me post GA with https://mirror.openshift.com/libra/rhui-rhel-server-7-ose-4.0/
-  - name: rhel-server-7-ose-4.0
-    baseurl: https://mirror.openshift.com/enterprise/all/4.0/latest/x86_64/os
+  # TODO: Replace me post GA with https://mirror.openshift.com/libra/rhui-rhel-server-7-ose-3.11/
```

Comment:
Why would master branch point to 3.11 repos?

Reply:
because Openshift on Azure is not yet 4.0 compatible. so just to keep consistency what is in the head